activiti-core/activiti-engine/src/main/java/org/activiti/engine/impl/cfg/ProcessEngineConfigurationImpl.java

```diff
@@ -1095,17 +1095,19 @@ public void initDataSource() {
     }
   }
 
-  protected static Properties databaseTypeMappings = getDefaultDatabaseTypeMappings();
+  protected static Properties databaseVendorMappings = getDefaultDatabaseVendorMappings();
 
   public static final String DATABASE_TYPE_H2 = "h2";
   public static final String DATABASE_TYPE_HSQL = "hsql";
   public static final String DATABASE_TYPE_MYSQL = "mysql";
+    public static final String DATABASE_TYPE_MYSQL5 = "mysql5";
   public static final String DATABASE_TYPE_ORACLE = "oracle";
   public static final String DATABASE_TYPE_POSTGRES = "postgres";
   public static final String DATABASE_TYPE_MSSQL = "mssql";
+  public static final String DATABASE_TYPE_MARIADB = "mariadb";
   public static final String DATABASE_TYPE_DB2 = "db2";
 
-  public static Properties getDefaultDatabaseTypeMappings() {
+  public static Properties getDefaultDatabaseVendorMappings() {
```

Comment:
While the name makes sense, I don't think we should rename this public method. It will break any code using it.